src/avram/migrator/runner.cr

```diff
@@ -13,32 +13,35 @@ class Avram::Migrator::Runner
   end
 
   def self.db_name
-    (URI.parse(database_url).path || "")[1..-1]
+    credentials.database
```

Comment:
Wow this is so much nicer!

Reply:
yeah, we were calling `URI.parse` a lot, so I figure this has to be some sort of benefit this way.